README.md

```diff
@@ -84,8 +84,15 @@ _Column flex items set to `align-items: center` overflow their container_
   </tr>
   <tr valign="top">
     <td>
-      <a href="https://codepen.io/philipwalton/pen/xbRpMe">2.1.a</a> &ndash; <em>bug</em><br>
-      <a href="https://codepen.io/philipwalton/pen/ogYpVv">2.1.b</a> &ndash; <em>workaround</em>
+      2.1.a <em>bug</em> &ndash;
```

Comment:
If we're going to make this change, I think I might even prefer not linking to CodePen at all (especially since discovering that autoprefixer isn't working for IE10 anymore). If you want to link to the source code, we could keep the bug number linking to your new gh-page demos, and then we could put a separate link to the source afterward. What do you think?

So it would look something like this:

```html
<a href="https://akaustav.github.io/flexbugs/2.1.a-bug.html">2.1.a</a> &ndash; <em>bug</em> (<a href="docs/2.1.a-bug.html">source</a>)<br>
```

Though honestly, I'm not sure we need to link to the source since people can just view the source anyway (especially if we're able to generate the vendor prefixes as part of a build step).

Reply:
> If we're going to make this change, I think I might even prefer not linking to CodePen at all (especially since discovering that autoprefixer isn't working for IE10 anymore). If you want to link to the source code, we could keep the bug number linking to your new gh-page demos, and then we could put a separate link to the source afterward. What do you think?

I did not know how you'd feel if I had removed the links to the pens (which I am sure you had spent countless hours creating and perfecting). Hence, I had retained those pens. I am fine with not linking to CodePen at all.

> So it would look something like this:
> ```html
> <a href="https://akaustav.github.io/flexbugs/2.1.a-bug.html">2.1.a</a> &ndash; <em>bug</em> (<a href="docs/2.1.a-bug.html">source</a>)<br>
> ```

Done in my latest commits. Please take a look at the latest [README.md](https://github.com/akaustav/flexbugs/blob/master/README.md#flexbug-2) in my fork.

> Though honestly, I'm not sure we need to link to the source since people can just view the source anyway (especially if we're able to generate the vendor prefixes as part of a build step).

I think we were debating about the live editing feature which CodePen provided. Hence, I had kept a link to the source code - as the next best thing, I guess. We do NOT have to keep a link to the source code - I agree people can look at the source code from within the browser debugger tools (or GitHub - when they figure out how all code is organized inside the `docs` folder). At the moment, I have kept the link to the source as per your code suggestion. Please review and confirm if we should remove the link to the source.

Reply:
I think I lean slightly toward not linking to the source code (just to keep it consistent with how it is today), but I'm happy to defer to your judgement on that front. Whatever you think is best I'll go ahead with.

Reply:
@philipwalton Fair enough. I have removed the links to the source in the latest commit. Please take a look at the latest [README.md](https://github.com/akaustav/flexbugs/blob/master/README.md#flexbug-2) in my fork.